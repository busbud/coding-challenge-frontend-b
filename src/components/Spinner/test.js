import React from 'react';
import renderer from 'react-test-renderer';

import Component from './';

describe('Component', () => {
	it('should render', () => {
		const wrapper = renderer.create(<Component />).toJSON();
		expect(wrapper).toMatchSnapshot();
	});
});
