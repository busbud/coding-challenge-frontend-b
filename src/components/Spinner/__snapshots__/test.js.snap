// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should render 1`] = `
<svg
  className="spinner"
  version="1.1"
  viewBox="0 0 64 64"
>
  <path
    className="fidget"
    d="m32 2.2c-7.6 0-10.7 7.1-10.7 10.6 0 3.5 1 5.3 2.3 7 1.3 1.7 2.2 4.3-0.2 7.7-2.4 3.5-3.5 2.9-7.9 3.1-4.4 .1-10.5 3.9-10.5 11-0.1 7.1 6.4 10.4 10.4 10.5 4 .1 7.6-2.1 9.4-5.2 2.2-3.8 4-5.4 7.3-5.3 3.3 .1 5.1 1.5 7 4.9 1.8 3.4 5.5 5.6 9.6 5.6 4.2 0 10.5-2.8 10.4-10.8-0.1-8.1-6.6-10.3-10.3-10.5-3.7-0.2-6-0.1-8.1-3.3-2.1-3.2-1.2-5.9-0.1-7.5 1.1-1.7 2.3-3.2 2.3-7.1C42.8 8.7 39.6 2.2 32 2.2zm0.1 4.3c3.6 0 6.5 2.9 6.5 6.5 0 3.6-2.9 6.5-6.5 6.5-3.6 0-6.5-2.9-6.5-6.5 0-3.6 2.9-6.5 6.5-6.5zm0 18.8c3.6 0 6.5 2.9 6.5 6.5 0 3.6-2.9 6.5-6.5 6.5-3.6 0-6.5-2.9-6.5-6.5 0-3.6 2.9-6.5 6.5-6.5zm-16.3 9.4c3.6 0 6.5 2.9 6.5 6.5 0 3.6-2.9 6.5-6.5 6.5-3.6 0-6.5-2.9-6.5-6.5 0-3.6 2.9-6.5 6.5-6.5zm32.7 0c3.6 0 6.5 2.9 6.5 6.5 0 3.6-2.9 6.5-6.5 6.5-3.6 0-6.5-2.9-6.5-6.5 0-3.6 2.9-6.5 6.5-6.5z"
  />
</svg>
`;
